src/main/java/com/snowflake/kafka/connector/internal/SnowflakeConnectionService.java

```diff
@@ -188,12 +188,15 @@
    * @param content file content
    * @param stageName stage name
    */
+  @Deprecated
   void put(String stageName, String fileName, String content);
```

Comment:
Is it possible to remove this entirely?

Reply:
Yes, we wont need this. Will remove. I kept it to see how old and new one performed in terms of latency.

Reply:
Will remove in future changes.